addons/project/tests/test_project_subtasks.py

```diff
@@ -356,3 +378,27 @@ def test_get_all_subtasks(self):
             self.assertEqual(parent_id, task.id, "The key should be the common ancestor")
             self.assertEqual(set(subtask_ids), set(all_subtasks.ids),
                              "All subtasks linked to the common ancestor should be returned by _get_subtask_ids_per_task_id method")
+
+    def test_write_project_should_propagate_to_no_display_subtasks(self):
+        task = self.env['project.task'].create({
+            'name': 'proute visible',
+            'project_id': self.project_goats.id,
+            'child_ids': [
+                Command.create({
+                    'name': 'sous-proute invisible',
+                    'project_id': self.project_goats.id,
+                }),
+                Command.create({
+                    'name': 'sous-proute visible',
+                    'project_id': self.project_goats.id,
+                    'display_in_project': True,
+                }),
+            ],
+        })
```

Comment:
Maybe find another task name 🤣 

```suggestion
        task = self.env['project.task'].create({
            'name': 'Task visible',
            'project_id': self.project_goats.id,
            'child_ids': [
                Command.create({
                    'name': 'Sub-Task invisible',
                    'project_id': self.project_goats.id,
                }),
                Command.create({
                    'name': 'Sub-Task visible',
                    'project_id': self.project_goats.id,
                    'display_in_project': True,
                }),
            ],
        })
```

Otherwise, we will soon have a test with those task names:

```suggestion
        task = self.env['project.task'].create({
            'name': '💩 🙉',
            'project_id': self.project_goats.id,
            'child_ids': [
                Command.create({
                    'name': '💩💩 🙈',
                    'project_id': self.project_goats.id,
                }),
                Command.create({
                    'name': '💩💩 🙉',
                    'project_id': self.project_goats.id,
                    'display_in_project': True,
                }),
            ],
        })
```